apps/api/src/shared/database/seeds/user-seeder.ts

```diff
@@ -1,22 +1,67 @@
 import type { Dictionary, EntityManager } from '@mikro-orm/postgresql';
 import { Seeder } from '@mikro-orm/seeder';
 import { createUser } from '../factories/user.factory';
+import { ManagementClient } from 'auth0';
+import authConfig from 'config/auth.config';
+
+const { domain, managementApiIdentifier, clientId, clientSecret } =
+  authConfig();
+const managementClient = new ManagementClient({
+  domain,
+  clientId,
+  clientSecret,
+  audience: managementApiIdentifier,
+});
+
+const SEED_EMAIL_DOMAIN = 'example.org';
 
 export class UserSeeder extends Seeder {
   async run(em: EntityManager, context: Dictionary): Promise<void> {
+    const getUsersResponse = await managementClient.users.getAll({
+      q: `email:*${SEED_EMAIL_DOMAIN}`,
+      fields: 'user_id,email',
+    });
+    const identityProviderUsers = getUsersResponse.data;
+
+    const adminEmail = `admin@${SEED_EMAIL_DOMAIN}`;
     const pAdmin = createUser(em, {
-      email: 'admin@example.org',
+      email: adminEmail,
+      identityProviderId: await this.getIdentityProviderId(
```

Comment:
There is no need for this method to be async. Once we remove async keyword we can also omit await.